paz/optimization/losses/frustumpointnet_loss.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+import tensorflow as tf
+
+from paz.models.detection.model_util import NUM_HEADING_BIN, NUM_SIZE_CLUSTER, g_mean_size_arr, \
+    get_box3d_corners_helper, get_box3d_corners
```

Comment:
Why are these variables not as input to the loss. They should not exist as "global" variables.

Reply:
Also use full names i.e. what does "g" and "arr" mean in "g_mean_size_arr"?